app/javascript/home/components/SkillRow.vue

```diff
@@ -21,42 +18,54 @@ const props = defineProps({
     type: String,
     required: true,
   },
-  iconIdentifier: {
-    type: String,
-    required: false,
-    default: null,
-  },
-  plainIcon: {
-    type: Boolean,
-    default: true,
-  },
-  coloredIcon: {
-    type: Boolean,
-    default: true,
-  },
-  wordmarkedIcon: {
-    type: Boolean,
-    default: true,
-  },
 });
 
-const devIconClass = computed(() => {
-  const iconIdentifier = props.iconIdentifier || props.name.toLowerCase();
-  let iconClass = `devicon-${iconIdentifier}`;
-  if (props.plainIcon) iconClass = `${iconClass}-plain`;
-  if (props.wordmarkedIcon) iconClass = `${iconClass}-wordmark`;
-  if (props.coloredIcon) iconClass = `${iconClass} colored`;
+const svgContent = ref<null | string>(null);
 
-  return iconClass;
+const svgFiles = import.meta.glob('img/dev-icons/*.svg', {
+  query: '?raw',
+  import: 'default',
+});
+
+watchEffect(() => {
+  async function loadSvg() {
+    const fileName = props.name.toLowerCase().replace(/[^a-z0-9]+/g, '-');
+    const filePath = `../assets/images/dev-icons/${fileName}.svg`;
+
+    const svgFileForPath = svgFiles[filePath];
+
+    if (svgFileForPath) {
+      const fileSvgContent = await svgFileForPath();
+
+      if (typeof fileSvgContent === 'string') {
+        svgContent.value = fileSvgContent;
```

Comment:
I feel like I am trying to do something pretty simple -- inline some SVGs into the HTML -- but I guess it is not straightforward to do that with Vite.